### PR TITLE
Merging Develop into Test to apply ability to change workplace ID to Staging

### PR DESCRIPTION
**Pull request by aaron-russell**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -9,8 +9,6 @@ import { NestedRoutesService } from '@core/services/nested-routes.service';
 import { Angulartics2GoogleTagManager } from 'angulartics2/gtm';
 import { filter, take, takeWhile } from 'rxjs/operators';
 
-enableProdMode();
```
Comment (two-first-names) [inline src/app/app.component.ts:12]: Is this meant to being removed in this PR?

Comment (aaron-russell) [reply src/app/app.component.ts:12]: @chadtomkiss-madetech enabled it elsehwere as far as I remember

Comment (aaron-russell) [reply src/app/app.component.ts:12]: 410203505d9ca4f2060e2ff039a4c4d8c0ffd7d2

File: src/app/features/search/registration/registration.component.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    this.registrationForm = new FormGroup({
+      nmdsId: new FormControl(this.registration.establishment.nmdsId, [
+        Validators.required,
+        Validators.minLength(8),
+        Validators.maxLength(8),
+      ]),
+    });
+  }
+
+  get nmdsid() {
```
Comment (two-first-names) [inline src/app/features/search/registration/registration.component.ts:31]: This looks like it should be capitalised as `nmdsId`.

**Pull request merged**